api/desecapi/mail_backends.py

```diff
@@ -34,7 +35,9 @@ def _run_task(messages, debug, **kwargs):
         logger.warning('Sending queued email, details: %s', debug)
         kwargs.setdefault('backend', kwargs.pop('backbackend', MultiLaneEmailBackend.default_backend))
         with get_connection(**kwargs) as connection:
-            return connection.send_messages([dict_to_email(message) for message in messages])
+            content = connection.send_messages([dict_to_email(message) for message in messages])
```

Comment:
`send_messages` returns the number of emails sent (https://github.com/django/django/blob/master/django/core/mail/backends/smtp.py#L114), so let's call this variable `num_sent`